lib/process-services-cloud/src/lib/process-cloud/models/process-filter-cloud.model.ts

```diff
@@ -0,0 +1,52 @@
+/*!
+ * @license
+ * Copyright 2016 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export class ProcessInstanceQueryModel {
+    processDefinitionId: string;
+    appName: string;
+    state: string;
+    sort: string;
+    assignment: string;
+    order: string;
+
+    constructor(obj?: any) {
+        if (obj) {
+            this.appName = obj.appName || null;
+            this.processDefinitionId = obj.processDefinitionId || null;
+            this.state = obj.state || null;
+            this.sort = obj.sort || null;
+            this.assignment = obj.assignment || null;
+            this.order = obj.order || null;
+        }
+    }
+}
+export class ProcessInstanceFilterRepresentationModel {
+    name: string;
```

Comment:
can we add an Id property? In this way we can easy delete/save the filter in the future